common/src/main/java/org/jboss/eap/qe/ts/common/docker/Docker.java

```diff
@@ -0,0 +1,188 @@
+package org.jboss.eap.qe.ts.common.docker;
+
+import org.fusesource.jansi.Ansi;
+import org.junit.rules.ExternalResource;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+/**
+ * Intended to be used as a JUnit {@link org.junit.ClassRule ClassRule}.
```

Comment:
Example of usage is there which is great, but the basic description of what this class does is missing.

Reply:
What about having it in some sort of README file?